src/components/stretched-link-card/StretchedLinkCard.vue

```diff
@@ -125,6 +141,7 @@
                     :variant="theme === 'dark' ? 'on-dark' : 'primary'"
                     data-test="vs-stretched-link"
                     :disabled="disabled"
+                    :display-icon="businessSupport && isHomePage ? false : true"
```

Comment:
I think this could be removed and instead just pass in the linkType prop in "MegalinksLinkList" as "default" which should remove any icon